modules/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 
+option(ENABLE_ROSAPPS "Enable building the ReactOS extra applications" OFF)
+option(ENABLE_ROSTESTS "Enable building the ReactOS test suite" OFF)
+option(ENABLE_WALLPAPERS "Integrate wallpapers into the built ISOs" OFF)
+
```

Comment:
This looks like a stray commit that has already been merged into master.

Reply:
Please check the ability to use DPH after this patch. It seems to have broken this for me when I use the following:
gflags /p /enable regedit.exe /full. After this if I run regedit.exe, I get a memory access fault (0xC0000005). Thanks.

Reply:
I don't have such behaviour

Reply:
@Doug-Lyons if you enable break on first chance, you can investigate what it is.

Reply:
I am able to reproduce. Investigating.